scylla/src/transport/iterator.rs

```diff
@@ -9,6 +9,8 @@ use std::sync::Arc;
 use std::task::{Context, Poll};
 
 use futures::Stream;
+use scylla_cql::frame::request::query::PagingStateResponse;
+use scylla_cql::frame::response::result::RawRows;
```

Comment:
commit `treewide: propagate RawRows/new QueryResult` does not compile. Since last time, you renamed `RawRows` to `RawMetadataAndRawRows`.

Reply:
I was sure I ran `make ci` for all commits... my bad.